crypto/ec/ec_lib.c

```diff
@@ -475,6 +479,11 @@ const BIGNUM *EC_GROUP_get0_field(const EC_GROUP *group)
     return group->field;
 }
 
+int EC_GROUP_get_field_type(const EC_GROUP *group)
+{
+    return group->meth->field_type;
```

Comment:
Related: I think it is fine here not to check if group->meth is NULL before dereferencing it.